portfolio/src/main/webapp/index.html

```diff
@@ -21,6 +21,10 @@ <h1>My Page</h1>
       <menu> <a href="images.html">Album with photos</a> </menu>
 
       <menu> <a href="memes.html">Just some funny videos I watched recently😁</a> </menu>
+
+      <menu> <a href="/data">Java</a> </menu>
```

Comment:
As we discussed in the chat - this is part of a different change, but is being put together with this PR as we're not comfortable manipulating commits with git yet. Ideally this line would be part of a different branch, but we'll keep it in here for now.